mozilla_django_oidc/auth.py

```diff
@@ -99,7 +110,7 @@ def authenticate(self, code=None, state=None):
 
         # Validate the token
         token_response = response.json()
-        payload = self.verify_token(token_response.get('id_token'))
+        payload = self.verify_token(token_response.get('id_token'), **{'nonce': nonce})
```

Comment:
I think its less obscure to got with:

``
payload = self.verify_token(token_response.get('id_token'), nonce=nonce)
``